internal/k8s/controllers/reconciliation_test.go

```diff
@@ -179,7 +179,7 @@ var _ = AfterSuite(func() {
 	cancel()
 	err := testEnv.Stop()
 	Expect(err).NotTo(HaveOccurred())
-	Eventually(mgrDone.Load(), 5*time.Second, 200*time.Millisecond).Should(BeTrue())
+	Eventually(mgrDone.Load, 5*time.Second, 200*time.Millisecond).Should(BeTrue())
```

Comment:
Why this change?

Reply:
golangci-lint complains about
`
internal/k8s/controllers/reconciliation_test.go:182:2: ginkgo-linter: use a function call in Eventually. This actually checks nothing, because Eventually receives the function returned value, instead of function itself, and this value is never changed; consider using `Eventually(mgrDone.Load, 5*time.Second, 200*time.Millisecond).Should(BeTrue())` instead (ginkgolinter)
`

Reply:
Not sure you saw my previous comment, but can you move this to a separate commit (still in this pr)?

Reply:
ok